low-level/json_msgs/messages/base_msg.py

```diff
@@ -39,6 +41,21 @@
     def getJson(self):
         raise NotImplementedError("Subclasses should implement this!")
 
+    def add_common_resources(self, jsonMsg, request_type):
+        try:
+            if jsonMsg.get("message").get(

```

Comment:
Use KvPayload to do dictionary key validation in better way

Reply:
ok.. Done